pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/UserCreationRoute.java

```diff
@@ -0,0 +1,168 @@
+package org.broadinstitute.ddp.route;
+
+import java.time.Instant;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.validator.routines.EmailValidator;
+import org.apache.http.HttpStatus;
+import org.broadinstitute.ddp.constants.ErrorCodes;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.JdbiClient;
+import org.broadinstitute.ddp.db.dao.UserDao;
+import org.broadinstitute.ddp.db.dao.UserProfileDao;
+import org.broadinstitute.ddp.json.UserCreationPayload;
+import org.broadinstitute.ddp.json.UserCreationResponse;
+import org.broadinstitute.ddp.json.errors.ApiError;
+import org.broadinstitute.ddp.model.user.User;
+import org.broadinstitute.ddp.model.user.UserProfile;
+import org.broadinstitute.ddp.security.DDPAuth;
+import org.broadinstitute.ddp.service.participants.ParticipantsCreateService;
+import org.broadinstitute.ddp.service.participants.ParticipantsCreateService.ParticipantCreateError;
+import org.broadinstitute.ddp.util.ResponseUtil;
+import org.broadinstitute.ddp.util.RouteUtil;
+import org.broadinstitute.ddp.util.ValidatedJsonInputRoute;
+import spark.Request;
+import spark.Response;
+
+@Slf4j
+@AllArgsConstructor
+public class UserCreationRoute extends ValidatedJsonInputRoute<UserCreationPayload> {
+    @Override
+    protected int getValidationErrorStatus() {
+        return HttpStatus.SC_UNPROCESSABLE_ENTITY;
+    }
+
+    @Override
+    public Object handle(Request request, Response response, UserCreationPayload payload) throws Exception {
+        var auth = RouteUtil.getDDPAuth(request);
+
+        if (false && auth.isAuthenticated() == false) {
```

Comment:
Known `if` bypass here- currently working on testing out the authentication issues. Expect this line to be updated once I get I reliable source of DSM tokens.

Reply:
@MocanaAtBroad Code removed